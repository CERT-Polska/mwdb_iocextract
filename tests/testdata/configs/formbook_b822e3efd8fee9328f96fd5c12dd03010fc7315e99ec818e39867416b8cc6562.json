{"urls": [{"url": "www.crokve.com/k8g/"}], "decoy_domains": ["fqfou.info", "donots.click", "visionauti.com", "breakoutrealestate.com", "monarchestateplansmn.com", "madsyrup.info", "artemis-recouvrement.com", "lackiconsulting.com", "lacasitadelaspalmas.com", "mjcrittercreations.com", "aerotronasia.net", "yokld.com", "mediamazballon.com", "vidreaming.com", "starcitythryv.com", "gavbs6.com", "yunshuping.com", "beautifulplacesbynight.com", "obdha.info", "3birth.com", "pushittoprod.com", "csmhbz.com", "wepelekter.com", "cessecarrostunig2.online", "winanes.net", "xuanyu-tj.com", "new-sappslockmailsec7.info", "atenciontotalsalud.com", "adpcleaning.com", "pinkfoxxboutique.com", "creativesewandsew.com", "play1628betplace.com", "schooltells.com", "nigeriancocoaawards.com", "receivertv.com", "guideonwebs.com", "isqape.com", "thehighstatusclub.com", "youngstax.net", "vasengineracing.com", "matabishi.com", "sanfranciscosound.com", "thaigertrader.com", "whynotcosmetics.com", "foodov.com", "liya-bottle.com", "acaciawh.com", "xn--handkreissge-pcb.info", "thesweetblonde.com", "eurofashionshoes.com", "yournslc.com", "compactcolombo.win", "accstandard.com", "omnycom.com", "flooddrypros.com", "xiamenjijin.com", "appropriateexchange.com", "zipiki.com", "purduehyperloop.com", "eyesuncam.com", "practicalspanishonline.com", "reborncadiz.com", "maaxmind.com", "eljalifesciences.info"], "type": "formbook"}