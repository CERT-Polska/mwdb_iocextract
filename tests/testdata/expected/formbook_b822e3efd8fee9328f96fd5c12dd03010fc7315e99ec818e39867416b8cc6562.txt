NetLoc www.crokve.com//k8g/
